src/reactapp/src/i18n/index.js

```diff
@@ -1 +1,2 @@
 export { default as __ } from './__';
+export { default as __mt } from './__mt';
```

Comment:
are you keeping the old `__` for backwards compatibility?

Reply:
We are no longer using `__()` anywhere in the app. so the export in this file won't be an issue. The build js would not contain `__()` anymore.

But, it remains a redundant code in the codebase though. I will remove it then.

Reply:
could we rename `__mt` to `__` then?

Reply:
Sure

Reply:
I would prefer `__` over `__mt` too :)

Reply:
@wigman @schmengler yes, it is changed to `__()` now.